controllers/changetierrequest/changetierrequest_controller_test.go

```diff
@@ -224,6 +227,22 @@ func TestChangeTierFailure(t *testing.T) {
 			AllUserAccountsHaveTier(murtest.DefaultNSTemplateTier)
 		AssertThatChangeTierRequestHasCondition(t, cl, changeTierRequest.Name, toBeComplete(), toBeDeletionError("unable to delete ChangeTierRequest object 'request-name': error"))
 	})
+
+	t.Run("the change will fail because there is no corresponding UserSignup found", func(t *testing.T) {
```

Comment:
This test actually asserts that the reconcile fails if there is no owner label in the MUR, right? We need this test, so it's good. But we also need a test for the case when there is no UserSignup found. Or I'm missing something?